### DISCUSSION TITLE
Accessing Legally Purchased YouTube Content

### DISCUSSION THREAD

**Discussion by dplattipswich** [Tyrrrz/YoutubeExplode#675] [Q&A]
YouTube sells and rents copyrighted content, such as movies. In a browser, you can only stream them while logged into your Google/YouTube account. Is there any way, using this library, for an app you write to stream your legally purchased content? Perhaps by presenting credentials or using tokens in some way? Tx

**Comment by Tyrrrz** [comment]
You can pass credentials only in a way of cookies right now, by injecting your own `HttpClient` instance
